cli/commands/validate.go

```diff
@@ -0,0 +1,58 @@
+package commands
```

Comment:
I believe that `validatePack` should live in `pack/validate.go` but not in common place.

Reply:
Yes, I agree.

Fixed.